mail-filter/mimedefang/mimedefang-3.1.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Antispam, antivirus and other customizable filters for MTAs with Milter support"
+HOMEPAGE="http://www.mimedefang.org/"
+SRC_URI="https://mimedefang.org/releases/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="clamav +poll test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	acct-group/defang
+	acct-user/defang
+	dev-perl/Digest-SHA1
+	dev-perl/IO-stringy
+	dev-perl/MailTools
+	dev-perl/MIME-tools
+	dev-perl/Unix-Syslog
+	|| ( mail-mta/sendmail mail-mta/postfix )
+	mail-filter/libmilter:=
+	virtual/perl-MIME-Base64
+	clamav? ( app-antivirus/clamav )
+"
+
+DEPEND="
+	${RDEPEND}
+	test? (
+		dev-perl/Test-Class
+		dev-perl/Test-Most
+	)
+"
+
+src_prepare() {
+	eapply "${FILESDIR}/${PN}-2.72-ldflags.patch"
+	eapply "${FILESDIR}/${PN}-vendor.patch"
+	eapply "${FILESDIR}/${PN}-tests2.patch"
+	eapply_user
+}
```

Comment:
if you use a `PATCHES=( )` array you don't have to define `src_prepare` at all here.